crates/typst/src/eval/methods.rs

```diff
@@ -24,6 +25,32 @@ pub fn call(
             "lighten" => color.lighten(args.expect("amount")?).into_value(),
             "darken" => color.darken(args.expect("amount")?).into_value(),
             "negate" => color.negate().into_value(),
+            "kind" => match color {
+                Color::Luma(_) => vm.items.luma_func.into_value(),
+                Color::Rgba(_) => vm.items.rgb_func.into_value(),
+                Color::Cmyk(_) => vm.items.cmyk_func.into_value(),
+            },
+            "hex" => color.to_rgba().to_hex().into_value(),
```

Comment:
technically, it's hex rgba, but `rgba-hex` is meh and  `rgba().hex()` would only be possible if we had separate types which is overkill in my opinion. so I guess it's fine this way.

Reply:
yeah, and I guess people around the internet already assume `hex` to be RGBA anyway, so yeah